src/main/java/reiff/brickbreaker/ManyNetworks.java

```diff
@@ -0,0 +1,147 @@
+
+package reiff.brickbreaker;
+
+import basicneuralnetwork.NeuralNetwork;
+import lesser.brickbreaker.BrickBreakerComponent;
+import levy.brickbreaker.Ball;
+import levy.brickbreaker.Paddle;
+
+import java.awt.event.KeyEvent;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class ManyNetworks {
+
+    private final Ball ball;
+    private final Paddle paddle;
+    private final Controller controller;
+    private final BrickBreakerComponent view;
+
+    public ManyNetworks(Ball ball, Paddle paddle, Controller controller, BrickBreakerComponent view) {
+        this.ball = ball;
+        this.paddle = paddle;
+        this.controller = controller;
+        this.view = view;
+    }
+
+    //Here we are creating 1000 networks and storing them in an array.
+    public List<NeuralNetwork> generateNetworks() {
+
+        List<NeuralNetwork> neuralNetworkList = new ArrayList<>(1000);
+
+        for (int i = 0; i < 1000; i++) {
+            neuralNetworkList.add(new NeuralNetwork(1, 2, 4, 2));
+        }
+
+        return neuralNetworkList;
+    }
+    //now we have a list of 1000 network, one in each position.
+
+    public List<NeuralNetwork> createNextGeneration(List<NeuralNetwork> topPerformingNetworks) {
+
+        List<NeuralNetwork> nextGeneration = new ArrayList<>(1000);
+
+        for (int i = 0; i < topPerformingNetworks.size(); i++) {
```

Comment:
I don't think these two for loops end up with 1000 networks

Reply:
I see the problem, I am generating 90 networks out of the top 10. But once I am done merging each network with the others, what else is there left to merging in order to get to 1000?

Reply:
Change your approach. 

Merge one random network from the list with another random network from the list. 

Do that 1000 times

Reply:
Got it, thank you